core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1812,45 +1818,55 @@ public String toString() {
               .build();
   public static final PropertyKey MASTER_WORKER_TIMEOUT_MS =
       new Builder(Name.MASTER_WORKER_TIMEOUT_MS)
-          .setAlias(new String[]{"alluxio.master.worker.timeout.ms"})
+          .setAlias("alluxio.master.worker.timeout.ms")
           .setDefaultValue("5min")
           .setDescription("Timeout between master and worker indicating a lost worker.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
-  public static final PropertyKey MASTER_EXECUTOR_PARALLELISM =
-      new Builder("alluxio.master.executor.parallelism")
+  public static final PropertyKey MASTER_RPC_EXECUTOR_PARALLELISM =
+      new Builder(Name.MASTER_RPC_EXECUTOR_PARALLELISM)
```

Comment:
@ggezer let me know if you are good with this change to the following property names. e.g. "alluxio.master.executor.parallelism" -> "alluxio.master.rpc.executor.parallelism" to avoid confusion (there are multiple executors inside master). Also I updated the property description. PTAL if you are good